packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
Can you please mark this as 0.6.13 and update the `pubspec.yaml` as well?

Reply:
Done.

Reply:
> Can you please mark this as 0.6.13 and update the `pubspec.yaml` as well?

Why? This is a test-only change, so it doesn't require a version bump.

Reply:
Yes, the version bump should be reverted. Even the changelog is optional here; the repo policy changed somewhat recently to not require changelog entries for package-developer-only changes (such as changes that only affect tests).